gems/smithy/lib/smithy/plan.rb

```diff
@@ -12,10 +12,8 @@ def initialize(model, type, options = {})
       @options = options
       @service = find_service(model['shapes'])
 
-      Welds.load!(self)
-      Polishes.load!(self)
-      @welds = options[:skip_welds] ? [] : Welds.for(model)
-      @polishes = options[:skip_polishes] ? [] : Polishes.for(model)
+      Rivets.load!(self)
+      @rivets = Rivets.for(@service)
```

Comment:
I think we still need the ability to skip these (how are they getting skipped in tests now?)

Reply:
Talked offline